### DISCUSSION TITLE
Does typst support PDF layers?

### DISCUSSION THREAD

**Discussion by mainrs** [typst/typst#3975] [Q&A] [answered]
It is a really useful feature when designing documents to be printed out and viewed on the screen at the same time. 

> For example, you might want to add a watermark that’s invisible on screen but visible when the PDF is printed, or display a message on screen, but not on the printed version. Or perhaps you want the user to be able to change the language [1].

[1]: https://www.gdpicture.com/guides/gdpicture/Using%20PDF%20Layers%20or%20Optional%20Content%20Groups%20(OCG).html

**Comment by laurmaedje** [comment, marked as answer]
No, this feature is not currently supported.

**Reply by towoe** [reply]
Sorry if I get back to this answered discussion.
My question only slightly differs, it is not supported, but are layers somewhat on the roadmap?
Or is it foreseeable that this will not be implemented?
Thanks!
